net-mgmt/pfSense-pkg-nrpe/files/usr/local/pkg/nrpe2.inc

```diff
@@ -154,8 +154,14 @@ function nrpe2_custom_php_write_config() {
 		$sudo = (isset($cmd['sudo']) && is_executable($sudo_bin)) ? "{$sudo_bin} " : "";
 		$wcmd = !empty($cmd['warning']) ? "-w {$cmd['warning']}" : "";
 		$ccmd = !empty($cmd['critical']) ? "-c {$cmd['critical']}" : "";
-		if (is_executable("{$nagios_check_path}/{$cmd['command']}")) {
-			$cmds[] = "command[{$cmd['name']}]={$sudo}{$nagios_check_path}/{$cmd['command']} {$wcmd} {$ccmd} {$cmd['extra']}\n";
+
+		$path_to_command = "{$nagios_check_path}/{$cmd['command']}";
+		$is_custom_shell_command = ($cmd['command'] === 'Custom shell command');
```

Comment:
This variable is only used once, better to not set it and use the value directly if it isn't reused.